platform/engineering/frontend/eslint/eslint-testing.md

```diff
@@ -0,0 +1,57 @@
+# ESLint Rule Changes and Testing
+
+**Author:** Jhonny Gonzalez
+**Last Updated:** February 21, 2020
+**eslint version:** 4.18.2
+
+## Overview
+
+This is an initial process to test and deploy new rules for ESLint.
+
+## Objective
+
+To improve the code coverage and code quality in the repo by enforcing new ESLint rules. This also will help all VFS teams to write better code.
+
+## Background
+
+We installed ESLint a couple of years ago in the VA repo and hasn't been updated nor improved since then.
+
+Now, that VFS teams will have an increased ownership on their pull request reviews, this will help them getting a better judgement as well as facilitate the review process.
+
+# Specifics
+
+## ESLintin Procedure
+
+The ESLinting rules in CircleCI will be used as an experimental staging area as we explore new rules and ways to communicate any issues that are discovered.
```

Comment:
```suggestion
The ESLint rules in CircleCI will be used as an experimental staging area as we explore new rules and ways to communicate any issues that are discovered.
```

Probably worth mentioning here that there will be two tasks in CircleCI. One that runs rules which are enforced, and one that runs rules which are experimental. It's this experimental workflow that we're talking about primarily here.

Reply:
Sounds good!